.readthedocs.yaml

```
@@ -0,0 +1,18 @@
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+version: 2
+
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.11"
+  # Build the Ansible docsite
+  commands:
+    - python -m venv "${READTHEDOCS_VIRTUALENV_PATH}"
+    - >-
+      "${READTHEDOCS_VIRTUALENV_PATH}"/bin/python -m pip install --exists-action=w -r requirements.in -c requirements.txt
+    - >-
+      "${READTHEDOCS_VIRTUALENV_PATH}"/bin/python -I build.py
+    - mkdir -pv "${READTHEDOCS_OUTPUT}"/html
+    - cp -rv output/* "${READTHEDOCS_OUTPUT}"/html
```

Did you forget to make use of the new script feature?
```suggestion
    - >-
      "${READTHEDOCS_VIRTUALENV_PATH}"/bin/python -I
      build.py "${READTHEDOCS_OUTPUT}"/html
```

---

Hey, yeah I was actually looking at that in the noxfile. Would you mind taking a look in my last commit? How can we pass that to the session to do something like `nox -s build -- "${READTHEDOCS_OUTPUT}"/html`?

---

And if we use nox then maybe we don't need the readthedocs venv.

---

Yes, just do this https://github.com/ansible/docsite/pull/182/files#r1296125285.